lib/eventFactory.js

```diff
@@ -175,7 +175,15 @@ function startBuild(config) {
  * @param  {Boolean}  [config.webhooks]                      If the create came from a webhook (pr or push) or not
  */
 function createBuilds(config) {
-    const { eventConfig, eventId, pipeline, pipelineConfig, changedFiles, webhooks } = config;
+    const {
+        eventConfig,
+        eventId,
+        pipeline,
+        pipelineConfig,
+        changedFiles,
+        webhooks,
+        prInfo
```

Comment:
Need to change jsdoc to include `prInfo`

Reply:
rename var to be more explicit